pkg/image/admission/imagepolicy/api/types.go

```diff
@@ -0,0 +1,129 @@
+package api
+
+import (
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/unversioned"
+)
+
+// ImagePolicyConfig is the configuration for controlling how images are used in the cluster.
+type ImagePolicyConfig struct {
+	unversioned.TypeMeta
+
+	// ExecutionRules determine whether the use of an image is allowed in an object with a pod spec.
+	// By default, these rules only apply to pods, but may be extended to other resource types.
+	ExecutionRules []ImageExecutionPolicyRule
+	// ConsumptionRules are applied when creating resources with pod specs.
+	ConsumptionRules []ImageConsumptionPolicyRule
+	// PlacementRules are applied when creating resources with pod specs.
+	PlacementRules []ImagePlacementPolicyRule
```

Comment:
I would say we'd disable this for 1.3

On Mon, Aug 1, 2016 at 11:50 AM, David Eads <notifications@github.com>
wrote:

> In pkg/image/admission/imagepolicy/api/types.go
> <https://github.com/openshift/origin/pull/8995#discussion_r73002199>:
>
> > +import (
> > +	"k8s.io/kubernetes/pkg/api"
> > +	"k8s.io/kubernetes/pkg/api/unversioned"
> > +)
> > +
> > +// ImagePolicyConfig is the configuration for controlling how images are used in the cluster.
> > +type ImagePolicyConfig struct {
> > +	unversioned.TypeMeta
> > +
> > +	// ExecutionRules determine whether the use of an image is allowed in an object with a pod spec.
> > +	// By default, these rules only apply to pods, but may be extended to other resource types.
> > +	ExecutionRules []ImageExecutionPolicyRule
> > +	// ConsumptionRules are applied when creating resources with pod specs.
> > +	ConsumptionRules []ImageConsumptionPolicyRule
> > +	// PlacementRules are applied when creating resources with pod specs.
> > +	PlacementRules []ImagePlacementPolicyRule
>
> I understand the point of this, but its strictly required for 1.3.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/8995/files/04c95de82a2919295f3a4f810c6c69389c5ccf47#r73002199>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABG_p_ji79iWmNttIzsGG08GOKd6eP6Tks5qbhXHgaJpZM4IlDlb>
> .
>